src/deploy/table_device_policy.sql

```diff
@@ -0,0 +1,15 @@
+BEGIN;
+
+GRANT SELECT, INSERT, UPDATE, DELETE ON TABLE maevsi.device TO maevsi_account;
+
+ALTER TABLE maevsi.device ENABLE ROW LEVEL SECURITY;
+
+CREATE POLICY device_new ON maevsi.device WITH CHECK (
+  TRUE
+);
+
+CREATE POLICY device_existing ON maevsi.device USING (
+  created_by = maevsi.invoker_account_id()
+);
```

Comment:
@sthelemann I'm proposing a new policy definition style here which focuses more on the difference between `USING` and `WITH CHECK` policies, applying them to all CRUD operations as I've noticed that we mostly duplicate policies for each CRUD operation and potentially missed some definitions on `UPDATE`.